Parser/Chat/Parser.ts

```diff
@@ -4,29 +4,65 @@ import {
 } from 'crt-terminal';
 import messages from '../../Messages/Messages';
 import { getOpenKey, signData, decryptMessage } from "../WEB3/chat/metamaskAPI"
-import { makeRequest, getWhitelist, encryptMessage, User } from "./utils"
+import { makeRequest, getWhitelist, encryptMessage, checkAccounts, GTON_MINIMUM, ListItem } from "./utils"
```

Comment:
Let's make naming style universal, say gtonChatThreshold here (we need to be more specific what the figure stands for)